command/alloc_status.go

```diff
@@ -133,6 +133,10 @@ func (c *AllocStatusCommand) Run(args []string) int {
 	var statsErr error
 	var stats *api.AllocResourceUsage
 	stats, statsErr = client.Allocations().Stats(alloc, nil)
+	if statsErr != nil {
+		c.Ui.Output("")
+		c.Ui.Error(fmt.Sprintf("couldn't retreive stats: %v", statsErr))
```

Comment:
`couldn't retrieve stats (ensure Client.Advertise.HTTP is set): %v`

You spelled retrieve wrong too.

Can you add similar error message to node-status